src/EnergyPlus/NodeInputManager.cc

```diff
@@ -1008,223 +974,209 @@ void CalcMoreNodeInfo(EnergyPlusData &state)
-            state.dataLoopNodes->MoreNodeInfo(iNode).SpecificHeat = Cp; // always fill since cp already always being calculated anyway
-            state.dataLoopNodes->MoreNodeInfo(iNode).WetBulbTemp = 0.0;
-            state.dataLoopNodes->MoreNodeInfo(iNode).RelHumidity = 100.0;
-        } else if (state.dataLoopNodes->Node(iNode).FluidType == DataLoopNode::NodeFluidType::Steam) {
-            if (state.dataLoopNodes->Node(iNode).Quality == 1.0) {
-                SteamDensity = GetSatDensityRefrig(state,
-                                                   fluidNameSteam,
-                                                   state.dataLoopNodes->Node(iNode).Temp,
-                                                   state.dataLoopNodes->Node(iNode).Quality,
-                                                   state.dataLoopNodes->Node(iNode).FluidIndex,
-                                                   RoutineName);
-                EnthSteamInDry = GetSatEnthalpyRefrig(state,
-                                                      fluidNameSteam,
-                                                      state.dataLoopNodes->Node(iNode).Temp,
-                                                      state.dataLoopNodes->Node(iNode).Quality,
-                                                      state.dataLoopNodes->Node(iNode).FluidIndex,
-                                                      RoutineName);
-                state.dataLoopNodes->MoreNodeInfo(iNode).VolFlowRateStdRho = state.dataLoopNodes->Node(iNode).MassFlowRate / SteamDensity;
-                state.dataLoopNodes->MoreNodeInfo(iNode).ReportEnthalpy = EnthSteamInDry;
-                state.dataLoopNodes->MoreNodeInfo(iNode).WetBulbTemp = 0.0;
-                state.dataLoopNodes->MoreNodeInfo(iNode).RelHumidity = 0.0;
-            } else if (state.dataLoopNodes->Node(iNode).Quality == 0.0) { // The node has condensate water through it
-                state.dataLoopNodes->MoreNodeInfo(iNode).VolFlowRateStdRho = state.dataLoopNodes->Node(iNode).MassFlowRate / RhoWaterStdInit;
-                state.dataLoopNodes->MoreNodeInfo(iNode).ReportEnthalpy =
-                    CPCW(state.dataLoopNodes->Node(iNode).Temp) * state.dataLoopNodes->Node(iNode).Temp;
-                state.dataLoopNodes->MoreNodeInfo(iNode).WetBulbTemp = 0.0;
-                state.dataLoopNodes->MoreNodeInfo(iNode).RelHumidity = 0.0;
+            thisMoreNodeInfo.VolFlowRateStdRho = thisNode.MassFlowRate / rhoStd;
+            thisMoreNodeInfo.VolFlowRateCrntRho = thisNode.MassFlowRate / rho;
+            thisMoreNodeInfo.Density = rho;
+            thisMoreNodeInfo.ReportEnthalpy = Cp * thisNode.Temp;
+            thisMoreNodeInfo.SpecificHeat = Cp; // always fill since cp already always being calculated anyway
+            // thisMoreNodeInfo.WetBulbTemp = 0.0; for water nodes, initialized to zero
+            thisMoreNodeInfo.RelHumidity = 100.0;
```

Comment:
Do we really need to set this for a water node? Could initialize all to 100 if necessary? Or 1-time loop just for water nodes? Maybe not worth it.